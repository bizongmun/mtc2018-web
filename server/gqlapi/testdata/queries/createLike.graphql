mutation {
  createLike(input: { uuid: "aaaa", sessionId: 12 }) {
    clientMutationId
    like {
      id
      session {
        id
        title
      }
    }
  }
}
