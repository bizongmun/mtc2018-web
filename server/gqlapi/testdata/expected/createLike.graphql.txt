--- createLike.graphql
mutation {
  createLike(input: { uuid: "aaaa", sessionId: 12 }) {
    clientMutationId
    like {
      id
      session {
        id
        title
      }
    }
  }
}


--- Result
{"data":{"createLike":{"clientMutationId":null,"like":{"id":"Like:1","session":{"id":"Session:12","title":"Introduction to the Corporate Solutions Engineering"}}}}}

--- Result (formatted)
{
  "data": {
    "createLike": {
      "clientMutationId": null,
      "like": {
        "id": "Like:1",
        "session": {
          "id": "Session:12",
          "title": "Introduction to the Corporate Solutions Engineering"
        }
      }
    }
  }
}

